app/views/idv/review/new.html.slim

```diff
@@ -3,6 +3,9 @@
 h1.h3 = t('idv.titles.session.review')
 
 p = t('idv.messages.sessions.review_message')
+a(target="_blank" href= "https://login.gov/security/")
+  = t('idv.messages.sessions.read_more_encrypt')
+
```

Comment:
it's hidden deep inside the CI noise, but the slim linter got cranky at this blank line:

```
app/views/idv/review/new.html.slim:9 [W] EmptyLines: Extra empty line detected
```

You can run `bundle exec slim-lint` locally